digid_eherkenning/templates/digid_eherkenning/post_binding.html

```diff
@@ -1,13 +1,16 @@
 {% load static %}
 <html>
     <head>
-        <title>Ga naar DigID</title>
+        <title>U wordt doorgestuurd</title>
+        <link rel="stylesheet" type="text/css" href='{% static "login_submit.css" %}'>
     </head>
     <body>
-        <script src="{% static 'login_submit.js' %}"></script>
+      <script src="{% static 'login_submit.js' %}"></script>
+      <div class="submit_form">
```

Comment:
can't this classname be applied directly to the `form` tag instead of adding another wrapper div?

Reply:
Joeri mentioned not being 100% sure if hiding the form directly would work, done but will require double-checking on a DigiD preproduction environment to be sure.